### PR TITLE
Andrei/agora 1921 turn on the list of delegates for etherfi

### PR DESCRIPTION
**Pull request by andreitr**
image

- Added "Proposals" button to the header
- Moved lowercased all ether.fi references
- Tested token delegation and redelagation

---

## PR-Codex overview
The focus of this PR is to update UI content and functionality based on the `Tenant` configurations. 

### Detailed summary
- Updated UI content in various components based on `Tenant` configurations
- Added `href` property to `UIPage` type
- Modified `DelegateCardList` component to filter delegates based on allowed list
- Enhanced Navbar functionality to use `ui` object from `Tenant`
- Updated `TenantUI` configurations with delegate addresses and page titles

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: src/components/Header/Navbar.jsx [hunk 1]
```diff
@@ -2,50 +2,48 @@
...
-          >
-            Proposals
-          </HeaderLink>
-        )}
+    <div className={`flex flex-row ${styles.main_nav}`}>
+      {hasProposals && (
+        <HeaderLink
+          href={hasProposalsHref ? ui.page("proposals")?.href : "/"}
+          target={hasProposalsHref ? "_blank" : "_self"}
+          isActive={pathname.includes("proposals") || pathname === "/"}
```
Comment (andreitr) [inline src/components/Header/Navbar.jsx:21]: Enabling our nav to work with external links. Not loving this but Ether.fi needs to link to proposal snapshot until our governor is in place.

**Review by andreitr — COMMENTED** [review]
Left a few comments to make it easier on the reviewer :)

File: src/components/Metrics/DAOMetricsHeader.jsx [hunk 2]
```diff
@@ -69,7 +69,7 @@ export default function DAOMetricsHeader({ metrics }) {
                       side="bottom"
                       sideOffset={3}
                     >
-                      <span>Total amount of OP in existence</span>
+                      <span>Total amount of {token.symbol} in existence</span>
                     </HoverCardContent>
```
Comment (jnmclarty) [inline src/components/Metrics/DAOMetricsHeader.jsx:73]: Just want to confirm that `.symbol` is declared for both op and ether.fi.  From line 63, I trust it must have been set for OP before(?), but I don't see a `symbol = ETHFI` in this PR.

File: src/lib/tenant/tenantUIFactory.ts [hunk 3]
```diff
@@ -155,6 +171,7 @@ const optimismUI = new TenantUI({
   hero: optimismHero,
 
   delegates: {
+    allowed: [],
     advanced: [],
```
Comment (jnmclarty) [inline src/lib/tenant/tenantUIFactory.ts:175]: Do you need to set an empty array for `ensUI`'s equivalent of a `delegates`?  Or, is optimism and ether.fi the only two that have non-null delegate-information?

**Review by jnmclarty — APPROVED** [review]
I took a look; I doubt I'm qualified yet to actually approve.  Consider getting one other reviewer if you think there is high risk.

File: src/components/Metrics/DAOMetricsHeader.jsx (hunk 2 above)
Comment (andreitr) [reply src/components/Metrics/DAOMetricsHeader.jsx:73]: Correct, the token symbol is set for each tenant.

File: src/lib/tenant/tenantUIFactory.ts (hunk 3 above)
Comment (andreitr) [reply src/lib/tenant/tenantUIFactory.ts:175]: That parameter is optional, I will remove it for OP. Good catch.

**Pull request merged**